proto/regen/ecocredit/basket/v1/tx.proto

```diff
@@ -21,6 +21,12 @@ service Msg {
   // Take takes credits from a basket starting from the oldest
   // credits first.
   rpc Take(MsgTake) returns (MsgTakeResponse);
+
+  // UpdateBasketFee is a governance method that allows for updation of fees to
+  // be used for the basket creation fee.
+  //
+  // Since Revision 1
+  rpc UpdateBasketFee(MsgUpdateBasketFee) returns (MsgUpdateBasketFeeResponse);
```

Comment:
```suggestion
  // UpdateBasketFees is a governance method that allows for updation of fees to
  // be used for the basket creation fee.
  //
  // Since Revision 1
  rpc UpdateBasketFees(MsgUpdateBasketFees) returns (MsgUpdateBasketFeesResponse);
```